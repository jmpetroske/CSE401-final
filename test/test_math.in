$somevar = 5 * 10px;

.my-header {
    $pad = 5px;

    font-size: 2 * 12px;
    padding: (2 * $pad) 0 0 (2 * $pad);
    height: $somevar / 2px;
}