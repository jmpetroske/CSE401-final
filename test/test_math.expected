.my-header {
font-size: 24px;
padding: 10px 0 0 10px;
height: 25px;
}
